src/NHSD.BuyingCatalogue.Organisations.Api/Models/Organisation.cs

```diff
@@ -1,22 +1,37 @@
 ﻿using System;
+using Newtonsoft.Json;
 
 namespace NHSD.BuyingCatalogue.Organisations.Api.Models
 {
     public sealed class Organisation
     {
+        private Address _location;
+
         public Guid Id { get; set; }
 
         public string Name { get; set; }
 
         public string OdsCode { get; set; }
 
+        public string PrimaryRoleId { get; set; }
+
+        public string Address { get; set; }
+
+        public bool CatalogueAgreementSigned { get; set; }
+
         public DateTime LastUpdated { get; set; }
 
-        public Organisation(Guid id, string name, string odsCode)
+        public Address Location
         {
-            Id = id;
-            Name = name;
-            OdsCode = odsCode;
+            get
+            {
+                if (Address != null)
+                {
+                    _location = JsonConvert.DeserializeObject<Address>(Address);
```

Comment:
Ideally the domain model should have no idea how the data is stored. Adding the JSON and backing field will couple the domain with our infrastructure.

Here is a resource for reference (we are using an Address object rather than a dictionary).
[Use Json conversion with EF Core](https://www.jerriepelser.com/blog/store-dictionary-as-json-using-ef-core-21/)